### DISCUSSION TITLE
New Relic support?

### DISCUSSION THREAD

**Discussion by memic84** [nitrojs/nitro#1245] [Q&A]
Anyone experience with Nuxt/Nitro for New Relic?

The Node.js (APM) agent doesn't seem to work with Nitro. Does anybody know if we need to add something else besides the default setup in NR?

<img width="1164" alt="image" src="https://github.com/unjs/nitro/assets/3043840/9163f2f5-b9fd-4a5c-994f-a62796ae7cd3">

https://docs.newrelic.com/docs/apm/agents/nodejs-agent/troubleshooting/troubleshoot-your-nodejs-installation/

**Comment by tnabil** [comment]
Hi @memic84, did you ever find a solution for this?

**Comment by DavidDeSloovere** [comment]
I would love to see OTEL support in Nitro. 
With OTEL you can send to many destinations, including New Relic.
